src/App/App.xaml.cs

```diff
@@ -407,18 +407,6 @@ private async Task TryClearCiphersCacheAsync()
         private async Task LockedAsync(bool autoPromptBiometric)
         {
             await _stateService.PurgeAsync();
-            if (autoPromptBiometric && Device.RuntimePlatform == Device.iOS)
```

Comment:
For the moment let's leave this iOS if block in place.  I was able to trigger a race condition resulting in a double-prompt and everything went sideways (though I've only been able to do it once).  Go ahead and remove the Android else block though, as that seems to achieve the desired result.